src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -214,7 +214,7 @@ public function load(array $configs, ContainerBuilder $container)
             }
 
             if (!\extension_loaded('intl') && !\defined('PHPUNIT_COMPOSER_INSTALL')) {
-                @trigger_error('Please install the "intl" PHP extension for best performance.', E_USER_DEPRECATED);
+                deprecated('', '', 'Please install the "intl" PHP extension for best performance.');
```

Comment:
what about `'symfony', 'latest'` for the general case

Reply:
actually, the package can still be `'symfony/framework-bundle'` here no?

Reply:
this message is generic - not specific to any package
implementation wise, displaying `Since symfony/framework-bundle : [...]` wouldn't make sense either.

Reply:
fair, but having the reference of the package telling us to use Intl instead would be generally useful :/

Reply:
> this message is generic - not specific to any package

on 2nd thought, that's fine as well :)